functions/index.js

```diff
@@ -60,6 +60,53 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.createNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var title = request.body.data.Title;
+        var des = request.body.data.Description;
+        var startDate = request.body.data.StartDate;
```

Comment:
startDate must be collected from the method which must be defined here. Not in ui.